test/GCHP/README.md

```diff
@@ -0,0 +1,82 @@
+# README for Integration Tests and Parallelization Tests
+
+## Overview:
+
+This directory contains scripts to submit automated GCHP integration tests, which will test the following processes:
+
+    - GCHP run directory creation
+    - GCHP code configuration and compilation
+    - GCHP execution
+
+Integration tests are short GCHP simulations.  Executables are compiled with all debugging options.  This should reveal any coding errors or run-directory configuration errors.
+
+## Files
+
+### Integration Test Scripts
+
+- `intTest.sh`
+  - Driver script to run GCHP integration tests
+- `intTestCreate.sh`
+  - Script to create GCHP run directories for integration tests
+- `intTestCompile.sh`
+  - Script to compile GCHP executables
+- `intTestExecute.sh`
+   - Script to run GCHP integration test simulations
+
+### Shared Scripts
+
+- `commonFunctionsForTests.sh`
+  - Link to `../shared/commonFunctionsForTests.sh`, which contains global variables and functions for the integration tests.
+
+## Running GCHP Integration Tests
+
+Before you submit any GCHP integration tests, please take a moment to:
+
+1. Verify that the `GCHP` superproject is checked out to the correct branch and commit.
+2. Run `git submodule update --init --recursive` in order to update all submodules.
+3. Verify that the `HEAD` commit of the `GEOS-Chem` submodule contains the code that you wish to test. (If not, then check out the proper branch.)
+4. Verify that the `HEAD` commit of the `HEMCO` submodule contains the code that you wish to test. (If not, then check out the proper branch.)
+
+### With the SLURM scheduler
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file -s -p partition
+```
+
+### With the LSF scheduler
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file -l -p partition
+```
+
+### Interactively at the command line
+
+```console
+$ cd test/GCClassic
+$ ./intTest.sh -d /path/to/test/dir -e /path/to/env-file
+```
```

Comment:
Three more cases of "GCClassic" snuck in here. They should be changed to "GCHP."

Reply:
This is all fixed in commit 75eccc02c4cb3513c6af3b542fa665d30dafe7e2